manifests/t/Tencent/TencentDocs/3.1.14/Tencent.TencentDocs.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.12 by Dumplings $debug=QUSU.CRLF.7-2-17.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Tencent.TencentDocs
+PackageVersion: 3.1.14
+InstallerType: nullsoft
+Scope: machine
+InstallerSwitches:
+  Upgrade: --updated
+UpgradeBehavior: install
+ReleaseDate: 2024-01-08
+Installers:
+- Architecture: x86
+  InstallerUrl: https://dldir1v6.qq.com/weiyun/tencentdocs/electron-update/release/3.1.14/TencentDocs3.1.14.ia32.30002.exe
+  InstallerSha256: 575220CE608FE6BAF10B8C408DB4F937358A84786966134760A7B22BF9058026
+  ProductCode: TDAppDesktop
+- Architecture: x64
+  InstallerUrl: https://dldir1v6.qq.com/weiyun/tencentdocs/electron-update/release/3.1.14/TencentDocs3.1.14.x64.30002.exe

```

Comment:
@SpecterShell,

Where did you find this URL? I couldn't find it from the PackageUrl, but instead got a different URL starting with `https://desktop.docs.qq.com/Installer/30001/3.1.13/TencentDocs-x86_64.exe? . . . `. But that was a signed URL that will probably expire soon.

Reply:
Their update API